lib/generation/generators/value_objects/generation_configuration/pb_generation_configuration.dart

```diff
@@ -135,8 +135,8 @@ abstract class GenerationConfiguration {
     await fileStructureStrategy.setUpDirectories();
   }
 
-  void _commitImports(
-      PBIntermediateNode node, String directoryName, String fileName) {
+  Future<void> _commitImports(
```

Comment:
Does this method need to be asynchronous? There are no awaits inside 🤔

Reply:
You are right, it is not necessary